### PR TITLE
Folder as a resource SD:#1850

### PR DESCRIPTION
**Pull request by Cmdv**

### CODE REVIEW

File: src/Quasar/Types.purs [hunk 1]
```diff
@@ -62,6 +63,12 @@ parseFile pt =
   <#> (Pt.rootDir </> _)
   # maybe (Left "Incorrect resource") pure
 
+parseAnyPath ∷ String → Either String (Pt.AbsPath Pt.Sandboxed)
+parseAnyPath pt = note "Incorrect resource" do
+    anyPath ← Pt.parsePath (const Nothing) (Just <<< Left) (const Nothing) (Just <<< Right) pt
```
Comment (kritzcreek) [inline src/Quasar/Types.purs]: we can reduce the indentation by two spaces here.

Comment (Cmdv) [reply src/Quasar/Types.purs]: 🕵️  will do :)

**Review by kritzcreek — APPROVED** [review]

**Comment by garyb** [comment]
Can I jump the queue again here - I have something I missed in the last PR that I need to get in before this breaks everything again :wink:

**Comment by Cmdv** [comment]
@garyb I don't think this is needed at all 😂

**Comment by garyb** [comment]
Closing as we indeed did not need it :)

**Pull request closed without merging**